### PR TITLE
Abandon context.xml and resort to fisco.properties

### PR DESCRIPTION
**Pull request by chaoxinhu**

### CODE REVIEW

File: src/test/java/com/webank/weid/BaseTest.java [hunk 1]
```diff
@@ -39,57 +37,52 @@
...
      */
     @Before
     public void setUp() {
 
-        privateKey = new BigInteger(toolConf.getPrivKey(), 16).toString();
+        privateKey = GenCredential.create(fiscoConfig.getPrivKey()).getEcKeyPair().getPrivateKey()
+            .toString();
+
+        authorityIssuerService = new AuthorityIssuerServiceImpl();
+        cptService = new CptServiceImpl();
```
Comment (chaoxinhu) [inline src/test/java/com/webank/weid/BaseTest.java]: Will this result in memory leak?

File: src/main/java/com/webank/weid/service/BaseService.java [hunk 2]
```diff
@@ -50,24 +53,59 @@
...
+        channelConnections.setConnectionsStr(Arrays.asList(fiscoConfig.getNodes().split(";")));
+        service.setAllChannelConnections(new ConcurrentHashMap<String, ChannelConnections>() {
+            {
+                put(fiscoConfig.getOrgId(), channelConnections);
+            }
+        });
+
+        // thread pool
+        ThreadPoolTaskExecutor pool = new ThreadPoolTaskExecutor();
+        pool.setBeanName("web3sdk");
```
Comment (chaoxinhu) [inline src/main/java/com/webank/weid/service/BaseService.java]: Move this and the build..() into TxnUtils()

File: src/main/java/com/webank/weid/service/BaseService.java [hunk 3]
```diff
@@ -50,24 +53,59 @@
...
     }
 
     private static boolean initWeb3j() {
-        Service service = context.getBean(Service.class);
+        logger.info("[BaseService] begin to init web3j instance..");
+        Service service = new Service();
+        service.setOrgID(fiscoConfig.getOrgId());
+        service.setConnectSeconds(Integer.valueOf(fiscoConfig.getWeb3sdkTimeout()));
+        // reconnect idle time 100ms
+        service.setConnectSeconds(100);
```
Comment (chenhaozx) [inline src/main/java/com/webank/weid/service/BaseService.java]: 这个100有什么依据

File: src/test/java/com/webank/weid/BaseTest.java (hunk 1 above)
Comment (chaoxinhu) [reply src/test/java/com/webank/weid/BaseTest.java]: fixed in codes below

File: src/main/resources/fisco.properties.tpl [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# Fisco-bcos blockchain node related properties
+
+# Common properties
+version=1.3
+
+privKey=bcec428d5205abe0f0cc8a734083908d9eb8563e31f943d760786edf42ad67dd
```
[the remark this answers is not in the document]
Comment (chaoxinhu) [reply src/main/resources/fisco.properties.tpl]: 这个是之前的保留行为，先留着

[the remark this answers is not in the document]
Comment (chaoxinhu) [reply src/main/resources/fisco.properties.tpl]: @chenhaozx

File: src/main/java/com/webank/weid/service/BaseService.java (hunk 3 above)
Comment (chaoxinhu) [reply src/main/java/com/webank/weid/service/BaseService.java]: removed

File: src/main/resources/fisco.properties.tpl (hunk 4 above)
[the remark this answers is not in the document]
Comment (chenhaozx) [reply src/main/resources/fisco.properties.tpl]: 不合适

**Comment by chaoxinhu** [comment]
Let's hold this for post-1.2.0 release

**Pull request closed without merging**